qt/scientific_interfaces/ISISReflectometry/GUI/MainWindow/IMainWindowView.h

```diff
@@ -29,6 +29,8 @@ class MainWindowSubscriber {
   virtual void notifyCloseBatchRequested(int) = 0;
   virtual void notifySaveBatchRequested(int) = 0;
```

Comment:
Can the `int` arguments be removed from these pure virtual functions?

Reply:
I don't think so - it looks like that's the batch index which I assume is used in the concrete implementation, and the signature here needs to match (although I haven't changed that in this PR so not sure how it's used off the top of my head).

Reply:
Yes, that makes sense!